src/Entity.js

```diff
@@ -214,22 +259,9 @@ exports = Class(function() {
 	 *         ~ returns the total distance moved by the two entities
 	 */
 
-	this.collidesWith = function(entity) {
-		return this.physics.collide(this, entity);
-	};
-
-	this.resolveCollidingStateWith = function(entity) {
```

Comment:
Thank @sjrct , this function was restored to support EntityPhysics.
If you use SATPhysics, you should use `response.overlap`.